com.sublimemerge.App.yaml

```diff
@@ -58,4 +59,15 @@ modules:
       - install apply_extra /app/bin
       - install sublime_merge.sh /app/bin/sublime_merge
       - install -Dm644 com.sublimemerge.App.appdata.xml /app/share/appdata/com.sublimemerge.App.appdata.xml
-      - install -Dm644 com.sublimemerge.App.png /app/share/icons/hicolor/128x128/apps/com.sublimemerge.App.png
\ No newline at end of file
+      - install -Dm644 com.sublimemerge.App.png /app/share/icons/hicolor/128x128/apps/com.sublimemerge.App.png
+  - name: git-lfs
```

Comment:
I would suggest moving the `git-lfs` module earlier in the list than the `sublime_merge` module, so that when we update sublime merge we can still use the cached build of git-lfs.